src/server/utilities/cspHeader/index.js

```diff
@@ -236,6 +236,9 @@ const directives = {
       'https://cdn.syndication.twimg.com', // Social Embeds
       'https://static.bbc.co.uk', // STY include
       'https://www.bbc.co.uk', // STY include
+      'https://passport-control.int.tools.bbc.co.uk/bookmarkletScript.js', // Passport bookmarklet - int
+      'https://passport-control.test.tools.bbc.co.uk/bookmarkletScript.js', // Passport bookmarklet - test
+      'https://passport-control.tools.bbc.co.uk/bookmarkletScript.js', // Passport bookmarklet - live
```

Comment:
Does setting an explicit resource (`/bookmarkletScript.js`) work in CSP? I'm curious, because I can't seem to find any documentation on it.

With these rules, would we expect `https://passport-control.tools.bbc.co.uk/bookmarkletScript.js` to pass and `https://passport-control.tools.bbc.co.uk/anotherScript.js` to fail?

Reply:
the resource itself might not matter, although it is more explicit when reading it at least. tested locally and this does still work, however we cant test whether another resource on this domain would work.

id be happy to clean this up in a later PR once we know this functionality is working.